tags/find/find.talon

```diff
@@ -0,0 +1,10 @@
+tag: user.find
+-
+hunt this: user.find("")
+hunt this (pace | paste):
+    user.find("")
+    sleep(25ms)
+    edit.paste()
+hunt this <user.text>: user.find(text)
+hunt next: user.find_next()
```

Comment:
I see these commands exist  (hunt next/previous) in find and replace too. Those duplicates should be removed.

(I can't comment on the correct file 'cause they didn't change)

Reply:
The latest commit should correct this.